src/calendar/index.tsx

```diff
@@ -69,6 +69,8 @@ export interface CalendarProps extends CalendarHeaderProps, DayProps {
   customHeader?: any;
   /** Allow selection of dates before minDate or after maxDate */
   allowSelectionOutOfRange?: boolean;
+  /** Max number of weeks to show on each month of month view */
```

Comment:
From the 'current' date. You need to mention that the weeks will start on the date passed to the 'current' prop and not from the first of the month.

Reply:
Yeah, let me change the name of the prop to make this more clear as well.